pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/generate/BaseStrategy.java

```diff
@@ -11,6 +11,9 @@ public class BaseStrategy implements Generator {
     public BaseStrategy(String fieldName, Object value) {
         this.fieldName = fieldName;
         this.value = value;
+        if (value instanceof String && ((String) value).isEmpty()) {
```

Comment:
verified during local testing that object passed is String instance and verified all dependencies of BaseStrategy